extensions/twilio/v2/actions/sendSms/config/fields.ts

```diff
@@ -16,6 +19,15 @@ export const fields = {
     description: 'The phone number you would like to send the text message to.',
     required: true,
   },
+  from: {
```

Comment:
Nitpick: move `from` above `recipient` as it's more natural to do things in this order: `from --> to --> message`.